server/user_account/tests.py

```diff
@@ -382,69 +213,34 @@ def test_logout_failure(self):
 
 
 class UpdateProfileTest(APITestCase):
+    fixtures = ["users.json"]
+
     # pylint: disable=too-many-instance-attributes
     def setUp(self):
         self.client = APIClient()
         self.url = "/user/"
 
-        organization = Organization.objects.create(org_name="Test")
         # Create each type of user that could be making the request
-        self.system_admin = CustomUser.objects.create(
-            user_name='system_admin',
-            email='system_admin@email.com',
-            password='password',
-            first_name='system',
-            last_name='admin',
-            role='SA',
-            is_active=True,
-            organization=organization)
-
-        self.manager = CustomUser.objects.create(
-            user_name='inventory1',
-            email='manager2@email.com',
-            password='123',
-            first_name='sy',
-            last_name='ad',
-            role='IM',
-            location='YUL',
-            is_active=True)
-
-        self.test_user = CustomUser.objects.create(
-            user_name='test',
-            email='test1@email.com',
-            password='password',
-            first_name='system',
-            last_name='admin',
-            role='SA',
-            is_active=True,
-            organization=organization)
-
-        self.stock_keeper = CustomUser.objects.create(
-            user_name='sk',
-            email='sk@email.com',
-            password='password',
-            first_name='stock',
-            last_name='keeper',
-            role='SK',
-            is_active=True,
-            organization=organization)
+        self.system_admin = CustomUser.objects.get(user_name="sa")
+        self.manager = CustomUser.objects.get(user_name="im")
+        self.stock_keeper = CustomUser.objects.get(user_name="sk")
 
         self.sys_admin_id = self.system_admin.id
-        self.test_user_id = self.test_user.id
+        # self.test_user_id = self.test_user.id
```

Comment:
Why are you commenting this out?

Reply:
sorry, it was a forgotten line to be removed. test_user was completely removed as it had no real purpose and it's attributes were the same as admin. Before deleting code I usually comment it out to verify nothing else is being used or if I want to go back to this code for whatever reason. Good catch!

Reply:
np!